app/assets/stylesheets/modules/top_navbar.scss

```diff
@@ -38,8 +37,36 @@
  }
 }
 
+.navbar {
+  max-height: 120px;
+
+  .header-bar {
```

Comment:
I'm not able to determine where this gets applied. Also should it live in `header_navbar.scss` if it is needed?

Reply:
Yep, it no longer appears to be needed. Removing.